### PR TITLE
Refactor dataset/signal endpoints into separate module

### PR DESCRIPTION
**Pull request by brilee**
Just reducing the volume of code in the router_dataset.py module in anticipation of adding a bunch more dataset editing (column rename, dataset rename) endpoints.

### CODE REVIEW

**Review by brilee — COMMENTED** [review]
oops wrong thread.

File: lilac/server.py [hunk 1]
```diff
@@ -113,6 +114,7 @@ def module_not_found_error(request: Request, exc: ModuleNotFoundError) -> JSONRe
 
 v1_router = APIRouter(route_class=RouteErrorHandler)
 v1_router.include_router(router_dataset.router, prefix='/datasets', tags=['datasets'])
+v1_router.include_router(router_dataset_signals.router, prefix='/datasets', tags=['datasets'])
```
Comment (nsthorat) [inline lilac/server.py:117]: do you need this since you import the router from the other file?

**Review by nsthorat — APPROVED** [review]

File: lilac/server.py (hunk 1 above)
Comment (brilee) [reply lilac/server.py:117]: fixed

Comment (dsmilkov) [reply lilac/server.py:117]: can you rerun run_server_dev.sh? It will auto-gen a new typescript file now

**Review by dsmilkov — APPROVED** [review]
Can you run_server_dev , it should auto-gen another typesript file now that you moved stuff to

File: lilac/server.py (hunk 1 above)
Comment (dsmilkov) [reply lilac/server.py:117]: also worth testing manually the UI

Comment (brilee) [reply lilac/server.py:117]: manually tested, and rerunning the gen didn't seem to change where the typescript file got generated?

**Comment by brilee** [comment]
nikhil says it's based on the URL base name so that's why no new file is created

File: lilac/server.py (hunk 1 above)
Comment (dsmilkov) [reply lilac/server.py:117]: ah, because you reused the same tag, which determines the file name, sgtm

**Pull request merged**